docs/wiki/framework/wardrobe-framework.md

```diff
@@ -0,0 +1,274 @@
+---
+layout: wiki
+title: Wardrobe Framework
+description: Explains the wardrobe system and how to make a compat.
+group: framework
+order: 5
+parent: wiki
+mod: ace
+version:
+  major: 3
+  minor: 20
+  patch: 0
+---
+
+
+## 1. Overview
+
+The Wardrobe addon gives the player the opportunity to change/modify their current wearables into different variants.
+
+For example, if a uniform has a normal variant and a "rolled-Up sleeves" variant, the player will be able to use ether an ACE Self Interaction or the CBA Context Menu in the inventory.
+
+Since there is no reliable, common patterns in terms of class inheritance, not even within the same DLC, each possible variant has to be defined individually within `configFile >> "ace_wardrobe"`.
+
+### 1.1 Components
+
+Components are an optional assistance to define a group of variants where a variant is made of one or more components.
+For example, a bandana with sunglasses would have the the bandana and the sunglasses as components, while the bandana and sunglasses only have themselves as a component.
+
+If the player switches from a variant with more components to a variant with less components, the surplus components will be deposited in their inventory. Vice Versa, if the player wants to change from a variant with less components to a variant with more components, they are required to have the missing components in their inventory, which will be removed on conversion.
+
+More examples can be found below.
+
+## 2. Config
+
+Only directly defined subclasses will be taken into account. A fully inherited subclass will be ignored and will not provide any interaction. The `ace_wardrobe` properties are to be found as a subclass of the item itself. Multiple base configs are designed at root of `configFile`.
+
+## 2.1 Properties
+All supported Properties can be found within the `ace_wardrobe_base` baseclass.
+
+| Class Property |  Data Type | Description |
+| -------------- |  ----------- | ----------- |
```

Comment:
Just for consistency, as the end result seems to still work just fine even without.
```suggestion
| Class Property | Data Type | Description |
| -------------- | --------- | ----------- |
```